editor/README_SETUP_LINUX.md

```diff
@@ -8,6 +8,10 @@ Download and install the latest JDK 11 release from either of these locations:
 
 * [Adoptium/Temurin](https://github.com/adoptium/temurin11-binaries/releases) - The Adoptium Working Group promotes and supports high-quality runtimes and associated technology for use across the Java ecosystem
 * [Microsoft OpenJDK builds](https://docs.microsoft.com/en-us/java/openjdk/download#openjdk-11) - The Microsoft Build of OpenJDK is a no-cost distribution of OpenJDK that's open source and available for free for anyone to deploy anywhere
+* or with:
+```
+> sudo apt-get install openjdk-11-jdk
```

Comment:
Which exact version do you get? It seems like it might be 11.0.15? https://packages.ubuntu.com/search?keywords=openjdk-11-jdk&searchon=names&suite=jammy&section=all

Reply:
Yes, indeed.

```
09:30:27 /usr/local/bin  $ whereis java
java: /usr/bin/java /usr/share/java /usr/share/man/man1/java.1.gz
09:31:06 /usr/local/bin  $ java --version
openjdk 11.0.15 2022-04-19
OpenJDK Runtime Environment (build 11.0.15+10-Ubuntu-0ubuntu0.20.04.1)
OpenJDK 64-Bit Server VM (build 11.0.15+10-Ubuntu-0ubuntu0.20.04.1, mixed mode, sharing)
```

Reply:
Ok, that is good. I think we should write "Download and install JDK 11.0.15 or later from either of these location"